Source/OutlineGlow/GlowSceneObjectContext.h

```diff
@@ -4,6 +4,8 @@
 #include <CS2/Classes/Color.h>
 #include <GameClasses/SceneObject.h>
 #include <Platform/Macros/IsPlatform.h>
+#include <MemoryPatterns/Linux/GlowSceneObjectPatternsLinux.h>
```

Comment:
This will cause build error on Windows - use `MemoryPatterns/PatternTypes/ClientPatternTypes.h` instead